packages/eas-cli/src/commands/env/pull.ts

```diff
@@ -24,7 +24,7 @@ export default class EnvironmentValuePull extends EasCommand {
     ...EASNonInteractiveFlag,
     path: Flags.string({
       description: 'Path to the result `.env` file',
-      default: '.env.local',
+      default: '.env.eas.local',
```

Comment:
TBH I would stick with `.env.local` as default, I believe that's what the average person expects, and `.env.local` works as well with Expo SDK by default but `.env.eas.local` doesn't.